CONTRIBUTING.md

```diff
@@ -37,193 +37,210 @@ To add a new page to the website, [create a new markdown file with YAML frontmat
-# device is physical devices
-# service is managed service offerings (SaaS/PaaS etc)
-# server-app are applications usually installed on the server-side
+# - os is for operating systems (and similar projects)
+# - db is for databases,
+# - app is for end-user applications,
+# - lang is for programming languages,
+# - framework is for application libraries, SDKs, frameworks...,
+# - device is for physical devices,
+# - service is for managed service offerings (SaaS/PaaS...),
+# - server-app is for applications usually installed on the server-side.
 category: os
 
-# Template to be used to generate a link for the release
-# __RELEASE_CYCLE__ will be replaced by the value of releaseCycle
-# __LATEST__ will be replaced by the value of latest
-# __CODENAME__ will be replaced by the optional codename
+# Simple Icons (https://simpleicons.org/) icon slug (mandatory).
+# If the icon is not available on Simple Icons, set it to `NA`.
+# As an example, https://simpleicons.org/?q=codemagic links to https://simpleicons.org/icons/codemagic.svg,
+# so the slug is `opensuse` (the SVG filename without extension).
+# A list of all slugs is also available on https://github.com/simple-icons/simple-icons/blob/develop/slugs.md.
+iconSlug: codemagic
+
+# Main URL for the page (mandatory).
+permalink: /timeturner
+
+# Alternate URLs that will redirect to the permalink (optional).
+# This is nice to let people use easier to remember URLs. For example, we redirect /golang to /go.
+alternate_urls:
+-   /hourglass
+
+# Command that can be used to check the current product version (optional).
+versionCommand: swish and flick
 
+# The more information link (optional).
+# If provided, this link is displayed after the product's description.
+# This link should contain information about the release policy and schedule. This is NOT the product URL!
+# Do not use a localized URL (such as one containing en-us) if possible.
+releasePolicyLink: https://nodejs.org/about/releases/
+
+# An image that shows a graphical representation of the releases (optional).
+# If provided, this image will be displayed at the top of the product's page.
+# This is not the product logo. Remove if you don't find a relevant image.
+releaseImage: https://raw.githubusercontent.com/nodejs/Release/main/schedule.svg?sanitize=true
+
+# Template to be used to generate a link for the releases (optional).
+# Available variables inside the template are:
+# - __RELEASE_CYCLE__: will be replaced by the value of `releaseCycle`,
+# - __LATEST__: will be replaced by the value of `latest`,
+# - __CODENAME__: will be replaced by the optional `codename`.
 # You can even use Liquid Templating inside the template, such as:
-# https://godotengine.org/article/maintenance-release-godot-{{"__LATEST__" | replace:'.','-'}}
-# Do not use a localized URL (such as one containing en-us) if possible
+# https://godotengine.org/article/maintenance-release-godot-{{"__LATEST__" | replace:'.','-'}}.
```

Comment:
lets not put any markings ( comma or dot ) at the end of this line to not to confuse people who might just copy and paste related line

Reply:
Yep, I also indented the line to make it more explicit it is an example.